spec/unit/resources/hab_build_spec.rb

```diff
@@ -7,12 +7,16 @@
       allow(File).to receive(:read).and_call_original
     end
```

Comment:
Not related to your PR, but I'm pretty sure you can remove the entire before block on lines 6-8.